delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/search/SearchParams.scala

```diff
@@ -119,6 +120,33 @@ object SearchParams {
         organization.forall(_ == resource.value.organizationLabel)
   }
 
+  /**
+    * Search parameters to filter data resources.
+    *
+    * @param deprecated the optional deprecation status of the project resources
+    * @param rev        the optional revision of the project resources
+    * @param createdBy  the optional subject who created the project resource
+    * @param updatedBy  the optional subject who updated the resource
+    * @param types      the resource types
+    * @param schema     the optional schema
+    */
+  final case class ResourceSearchParams(
```

Comment:
I think these will have to be ported to the view plugin; I would remove this for now.